src/Shared/Components/TabGroup/TabGroup.scss

```diff
@@ -0,0 +1,107 @@
+.tab-group {
+    list-style: none;
+
+    &__tab {
+        position: relative;
+
+        &::after {
+            content: '';
+            position: absolute;
+            bottom: 0;
+            left: 0;
+            width: 100%;
+            height: 2px;
+            background-color: transparent;
+            border-top-left-radius: 2px;
+            border-top-right-radius: 2px;
+        }
+
+        &--align-active-border::after {
+            bottom: -1px;
+        }
+
+        &:hover {
+            color: var(--B500);
+
+            .tab-group__tab__icon {
+                &--stroke {
+                    stroke: var(--B500);
+
+                    path {
+                        stroke: var(--B500);
+                    }
+                }
+
+                &--fill {
+                    fill: var(--B500);
+
+                    path {
+                        fill: var(--B500);
+                    }
+                }
+            }
+        }
+
+        &--active::after {
```

Comment:
let's not add any of fill and stroke when icon type is null for non-path elements